src/js/main.js

```diff
@@ -216,12 +220,8 @@ var SupportKit = Marionette.Object.extend({
             return $.Deferred().resolve(this.user);
         }
 
-        this.user.set(userInfo, {
-            parse: true
-        });
-
         this._throttledUpdate = this._throttledUpdate || _.throttle(this._updateUser.bind(this), 60000);
-        return this._throttledUpdate();
+        return this._throttledUpdate(userInfo);
```

Comment:
hmm I think we'll lose data if we throttle ... what happens if we do 3 update user in a row? only the last set of attributes if taken in account?